src/ServiceControl.Plugin.Nsb6.SagaAudit.AcceptanceTests/When_saga_changes_state.cs

```diff
@@ -33,7 +33,8 @@ public class When_saga_changes_state : NServiceBusAcceptanceTest
             Assert.True(context.TimeoutReceived);
 
             //SagaUpdateMessage Asserts
-            Assert.IsNotNullOrEmpty(context.MessagesReceived.First().SagaState, "SagaState is not set");
+            Assert.IsNotNull(context.MessagesReceived.First().SagaState, "SagaState is not set");
+            Assert.IsNotEmpty(context.MessagesReceived.First().SagaState, "SagaState is not set");
```

Comment:
With these messages being the same will it be possible to tell which one failed?